app/views/assessments/edit.html.erb

```diff
@@ -36,7 +53,164 @@
           </ul>
       <% end %>
 
-      <%= render "edit_#{params[:active_tab]}", f: f %>
+<h3 id="basic">Basic</h3>
+<hr>
+<%= f.text_field :name, disabled: true, class: "form-control",
+  help_text: "The unique (for this course) name of the assessment. Must be
+  lowercase alphanumeric (no punctuation). E.g.,
+  <kbd>malloclab</kbd>".html_safe %>
+
+<%= f.text_field :display_name,
+  help_text: "Name that will be displayed on the course home page. E.g.,
+  'Malloc Lab'" %>
+
+<div class="form-group">
+  <%= f.label :category_name, { :class=>"control-label" } %>
+  <%= f.collection_select :category_name, @course.assessment_categories, :to_s, :to_s, {selected: @assessment.category_name}, {class: "form-control"} %>
+  <p>or</p>
+  <%= text_field_tag :new_category, nil, placeholder: "New Category" %>
```

Comment:
`new_category` is not a field, so I can't seem to use f.text_field. Do you know how to get around it?